_posts/2020-08-09-monero-GUI-0.16.0.3-released.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Monero GUI 0.16.0.3 'Nitrogen Nebula' released"
+summary: GUI release of Monero 0.16.0.3
+tags: [releases]
+author: selsta
+---
+
+# Overview
+
+This is the v0.16.0.3 release of the Monero GUI software.
+
+[The latest CLI release notes can be found on the precedent blog post]({{ site.baseurl_root }}/2020/08/07/monero-0.16.0.3-released.html)
+
+Some highlights of this minor release are:
+
+- macOS app is now notarized by Apple
+- CMake improvments
+- Add support for IPv6 remote nodes
+- Add command history to Logs page
+- Add "Donate to Monero" button
+- Indicate probability of finding a block on Mining page
+- Minor bug fixes
+
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero-gui/compare/v0.16.0.2...v0.16.0.3), along with [the source code](https://github.com/monero-project/monero-gui/tree/v0.16.0.3).
+
+# Contributors for this Release
+
+This release was the direct result of 22 people who worked, largely unpaid and altruistically, to put out 71 commits containing 16626 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
+
+- luigi1111
+- xiphon
+- rating89us
+- selsta
+- glv2
+- xmrdsc
+
+A special thanks to translators
+
+8uDD4T, erciccione, Marius Tanislav, xmoreee, Lana, Michal Poupe, dskch83, anony\_xmr, C3Pool, Muge Niu, Anonymous, razorshaman909, Stavros, warraxxx, Georgios Apostolopoulos, bitbooz
```

Comment:
Could you remove my name from the list of translators and add it to the list above? I didn't really make translations.